contrib/win32build/anon-mingw.nsi.in

```diff
@@ -8,11 +8,11 @@
 !include "LogicLib.nsh"
 !include "FileFunc.nsh"
 !insertmacro GetParameters
-!define VERSION "0.4.9.0-alpha-dev"
+!define VERSION "0.4.10.0-alpha-dev"
```

Comment:
I believe it should also be reverted

Reply:
It is auto generated with `./scripts/maint/update_versions.py`, same as `APPROX_RELEASE_DATE` and comments, will update `AC_INIT()` is `configure.ac` and regenerate

Reply:
Updated to `0.4.9.0-alpha-dev`